open_city_profile/settings.py

```diff
@@ -214,3 +222,17 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Notification settings
```

Comment:
could you put these before `if "SECRET_KEY" not in locals():`, cause it's easier to keep that secret key generation logic at the end of the file, so you don't have to scroll over it every time you look through the settings

Reply:
Yes, definitely!